Makefile

```diff
@@ -58,6 +58,7 @@ TEST_REQUIRES_ROOT_PACKAGES=$(filter \
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress containerd-release
 
+BUILDTAGS=seccomp apparmor
```

Comment:
buildtags could be a parameter to the makefile.. probably should'nt be overriding here.